src/docs/platform-architecture-reference/openshift-network-policies.md

```diff
@@ -249,15 +266,67 @@ spec:
           port: 3306
 ```
 
-Once you have your network policy in place you'll need to set up some more pods to test. You can scale your deployment down to 1 pod to make things more straight forward. Keep in mind if you have any autoscalers in place.
+## Egress example - Allow only to specific Pod & Port
+
+Using the previous **Ingress** NetworkPolicy, create an **Egress** NetworkPolicy to allow only TO specific Pod & port.
```

Comment:
My understanding from https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-openshift-sdn-netpol-egress-policies was that egress policies can ONLY use to `ipBlock` and not `podSelector` or `namespaceSelector` ? Does other egress to options work?

Reply:
I think it's been supported? According to the [networking.k8s.io/v1 API reference doc](https://docs.openshift.com/container-platform/4.10/rest_api/network_apis/networkpolicy-networking-k8s-io-v1.html#spec-egress-to-2), these are can be used in the egress networkpolicy;

- .spec.egress[]
  - ports
  - ports[]
  - to
  - to[]
- .spec.egress[].ports[]
  - endPort
  - port
  - protocol
- .spec.egress[].to[]
  - ipBlock
  - namespaceSelector
  - podSelector

- .spec.egress[].to[].ipBlock
  - cidr
  - except

Reply:
It's in the API spec because other SDN technologies support full Egress. Like the NSX stuff. But the default SDN we are using does not, according to the release notes doc. It used to not support egress at all and just ignored it.

Reply:
OK, Thanks for clarifying. Removed that example. and added the note for **limited egress**.